### DISCUSSION TITLE
Miniconda3 is not working again

### DISCUSSION THREAD

**Discussion by Geozike** [giswqs/geog-312#65] [Q&A] [answered]
I had an issue with my wifi conduction. It goes off automatically. After trying to resolve the issue, my miniconda3 is not working again. Please what is problem and how do i resolve this?
![Problem 2](https://github.com/user-attachments/assets/21bb1412-14ef-410b-85ca-546b727cc750)
![Wifi connection problem](https://github.com/user-attachments/assets/30a83081-5163-4b8c-bee9-c9f60eacb32a)

**Comment by giswqs** [comment, marked as answer]
This is likely your own computer problem.
